portfolio/src/main/webapp/script.js

```diff
@@ -49,3 +49,24 @@ function openTab(evt, tabName) {
     evt.currentTarget.classList.add("active");
 }
 
+
+
+function hideButton(id) {
```

Comment:
Since if the element is already hidden this function will show it, maybe better to call this toggleVisibility or similar

Reply:
Also changed the name of this function in latest commit, agree that the naming convention should've addressed all of what it does, not just what I plan on using it for